raiwidgets/raiwidgets/error_analysis_dashboard_input.py

```diff
@@ -93,13 +100,21 @@ class and for the regression case a method of predict()
         :param num_leaves: The number of leaves of the surrogate tree
             trained on errors.
         :type num_leaves: int
+        :param sample_dataset: Used only if explanation is not specified for
```

Comment:
Might want to update this docstring like you did the other one

Reply:
done!